pycontrails/core/aircraft_performance.py

```diff
@@ -476,3 +490,62 @@ class AircraftPerformanceData:
     thrust: npt.NDArray[np.float_]
     engine_efficiency: npt.NDArray[np.float_]
     rocd: npt.NDArray[np.float_]
+
+
+# --------------------------------
+# Grid aircraft performance models
+# --------------------------------
+
+
+@dataclasses.dataclass
+class AircraftPerformanceGridParams(ModelParams):
+    """Parameters for :class:`AircraftPerformanceGrid`."""
+
+    #: Fuel type
+    fuel: fuel.Fuel = dataclasses.field(default_factory=fuel.JetA)
+
+    #: ICAO code designating simulated aircraft type.
+    #: Can be overridden by including ``aircraft_type`` attribute in source data
+    aircraft_type: str = "B737"
```

Comment:
At some point in the future, we should add an dictionary of all the ICAO codes. I'm often looking for this